slack-plugin/src/main/kotlin/slack/gradle/SlackBasePlugin.kt

```diff
@@ -347,4 +351,20 @@ internal class SlackBasePlugin : Plugin<Project> {
       }
     }
   }
+
+  private fun Project.configureBootstrapTasks() {
+    val rootTask = rootProject.tasks.named<CoreBootstrapTask>(CoreBootstrapTask.NAME)
```

Comment:
fyi at some point, I expect gradle _may_ make `rootProject` access "bad" too. I'm hoping they'll make some effort to provide a less-than-friction-full path, though.